src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+          class="images_item"
+          src="images/imac.jpeg"
+          alt="foto"
+       >
+
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K
+        Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card_code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">
+            Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__title">
+          Price:
+        </p>
+
+        <p class="card__value">
+           $2,199
+        </p>
```

Comment:
I guess this line of code can be fit into 1 line (it's not more than 80 symbols)
```suggestion
        <p class="card__value">$2,199</p>
```